event_product_logistics/views/delivery_receipt_report.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+     © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis -->
+
+<openerp>
+<data>
```

Comment:
Yes, it does not affect templates. When you manually edit them, its XMLID's `noupdate` becomes `True` and no more updates are applied.